exercism/hamming/hamming.src.js

```diff
@@ -0,0 +1,17 @@
+var Hamming = function() {};
+
+Hamming.prototype.compute =  function (a, b) {
```

Comment:
naming the variables `a` and `b` makes it hard to understand what the variables represent. Naming it something like `dna1` and `dna1` gives more context, and makes it readable